core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss

```diff
@@ -61,8 +61,13 @@
         background: $color-palette-primary-500;
     }
 
+    .p-tabview .p-tabview-nav li:not(.p-highlight):not(.p-disabled):hover .p-tabview-nav-link {
```

Comment:
Out of curiosity, is this selector `li:not(.p-highlight):not(.p-disabled):hover` needed?

Reply:
@fmontes Yeah, seems like we are combining more selectors than we should